setup.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+import os.path
+
+from setuptools.command.install import install as _install
+from setuptools import setup
+
+
+class install(_install):
+    def initialize_options(self):
+        _install.initialize_options(self)
+        self.extra_path = (self.distribution.metadata.name, 'import readline')
```

Comment:
this is the entire magic

if you want to understand .pth files I recommend my video on the topic: https://youtu.be/mzxQrgvuRFg